packages/genesys-spark-components/src/components/stable/gux-time-picker/tests/gux-time-picker.service.spec.ts

```diff
@@ -331,6 +331,71 @@ describe('gux-time-picker.service', () => {
     );
   });
 
+  describe('#getTimeDisplayValues with set boundaries', () => {
+    it(`should work as expected for 24h with 30 minute intervals and boundaries of 11:00-20:00`, async () => {
+      const minuteInterval = 30;
+      const clockType = '24h';
+      const min = '11:00';
+      const max = '20:00';
+      const expectedOutput = [
+        '11:30',
+        '12:00',
+        '12:30',
```

Comment:
Might just be worth adding a min of a single digit number to verify how that formatting works.

Reply:
Great catch. This has been added now.